src/orion/core/io/cmdline_parser.py

```diff
@@ -0,0 +1,229 @@
+# -*- coding: utf-8 -*-
+"""
+:mod:`orion.core.io.cmdline_parser` -- Parse command line arguments for Orion
+=============================================================================
+
+.. module:: cmdline_parser
+   :platform: Unix
+   :synopsis: Parsing and building of the command line input for using script
+
+Simplify the parsing of a command line by storing every values inside an `OrderedDict`
```

Comment:
Nitpicking; I think "Simplify" is irrelevant here.

Reply:
Would "Handles" be better here? There is also a "simple" inside the class description.